### DISCUSSION TITLE
New language

### DISCUSSION THREAD

**Discussion by haema5** [Cinnamon/kotaemon#259] [Q&A] [answered]
Hello! Please tell me how I can add a new language to the interface?

**Comment by taprosoft** [comment, marked as answer]
Please see this closed issue if you are aware of editing some source codes: https://github.com/Cinnamon/kotaemon/issues/163
Otherwise, we will provide options to use arbitrary language in the future.

**Reply by haema5** [reply]
Thank you! It worked.
